### PR TITLE
Test/network select

### PR DESCRIPTION
**Pull request by alinasytnik**
## Changelog

- Change 1
- Change 2
- Change 3

### Type of changes included:

- [x] Components
- [x] Business Logic
- [ ] Bug fixes
- [ ] Styling
- [ ] Code Refactor

### Clubhouse Tickets Related:

- Ticket 1
- Ticket 2
- Ticket 3

### Screenshots/Gifs:

*Place you changes screens

https://user-images.githubusercontent.com/33018206/188321510-a3005575-2a43-49ee-b101-b4bfe9c34319.mov

https://user-images.githubusercontent.com/33018206/188660117-594747a0-d0c8-4b30-befc-96cffcb887e4.mov

https://user-images.githubusercontent.com/33018206/188660665-9c766388-f469-4460-b925-3195ee636369.mov

hots here*

### Notes:

*Place special notes here*

### Tested on:

- [x] Chrome
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

**Review by karlamiesesfleek — APPROVED** [review]

File: tests/e2e/import-create-account.test.js [hunk 1]
```diff
@@ -124,7 +124,7 @@ describe('Import/Create', () => {
 
       const value = await page.evaluate((el) => el.textContent, plugBanner);
 
-      expect(value).toMatch('Alpha - 0.5.3');
+      expect(value).toMatch('Alpha - 0.6.0');
```
Comment (karlamiesesfleek) [inline tests/e2e/import-create-account.test.js]: Is there a way to avoid having this hardcoded? Like instead of toMatch have it greater than?

Comment (alinasytnik) [reply tests/e2e/import-create-account.test.js]: lol, sure
I don't know why I didn't do that 
Thanks 😊

**Review by karlamiesesfleek — COMMENTED** [review]
@alinasytnik I noticed that under Sonic testnet there are limitations, like user cannot access to certain tab and cannot search for tokens, and only can add Custom Token. Would you like at some point add testing for those scenarios also?
I would also include a send transaction just to confirm everything allowed under this network is working ok.

File: tests/e2e/import-create-account.test.js (hunk 1 above)
Comment (alinasytnik) [reply tests/e2e/import-create-account.test.js]: @alinasytnik I noticed that under Sonic testnet there are limitations, like user cannot access to certain tab and cannot search for tokens, and only can add Custom Token. Would you like at some point add testing for those scenarios also?
I would also include a send transaction just to confirm everything allowed under this network is working ok. - related to this comment

@karlamiesesfleek 

Yeah, I agree with you. I'm checking only the NFT tab on disabling and definitely should check it for all disabled functions. 
Regarding adding and sending tokens on sonic network, I'm currently working on it and will let you know when i finish it :)

**Review by karlamiesesfleek — APPROVED** [review]
beutiful!

**Review by karlamiesesfleek — APPROVED** [review]

**Pull request merged**